### PR TITLE
Changes and cleanups after review comments

### PR DESCRIPTION
**Pull request by pron**
---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Coleen Phillimore](https://openjdk.java.net/census#coleenp) (@coleenp - Committer) ⚠️ Review applies to 97d749e47f858c98bbc0bf88775e206dcd6a5eaa
 * [Robbin Ehn](https://openjdk.java.net/census#rehn) (@robehn - Committer) ⚠️ Review applies to 97d749e47f858c98bbc0bf88775e206dcd6a5eaa
 * [Erik Österlund](https://openjdk.java.net/census#eosterlund) (@fisk - Committer) ⚠️ Review applies to 9bb3e479b9a3d6123f73c87212eb5f5617a79ee9

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/loom pull/171/head:pull/171` \
`$ git checkout pull/171`

Update a local copy of the PR: \
`$ git checkout pull/171` \
`$ git pull https://git.openjdk.java.net/loom pull/171/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 171`

View PR using the GUI difftool: \
`$ git pr show -t 171`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/loom/pull/171.diff">https://git.openjdk.java.net/loom/pull/171.diff</a>

</details>

### CODE REVIEW

File: src/hotspot/cpu/aarch64/continuationEntry_aarch64.inline.hpp [hunk 1]
```diff
@@ -32,7 +32,9 @@
 #include "runtime/registerMap.hpp"
 
 inline frame ContinuationEntry::to_frame() const {
-  static CodeBlob* cb = CodeCache::find_blob(entry_pc());
+static CodeBlob* cb = CodeCache::find_blob_fast(entry_pc());
```
Comment (coleenp) [inline src/hotspot/cpu/aarch64/continuationEntry_aarch64.inline.hpp:35]: looks like needs 2 indent.

File: src/hotspot/cpu/aarch64/frame_aarch64.cpp [hunk 2]
```diff
@@ -159,6 +159,11 @@ bool frame::safe_for_sender(JavaThread *thread) {
       sender_pc = pauth_strip_verifiable((address) *(sender_sp-1), (address)saved_fp);
     }
 
+    if (Continuation::is_return_barrier_entry(sender_pc)) {
+      frame s = Continuation::continuation_bottom_sender(thread, *this, sender_sp);
+      sender_sp = s.sp();
+      sender_pc = s.pc();
+    }
```
Comment (coleenp) [inline src/hotspot/cpu/aarch64/frame_aarch64.cpp:166]: this function is too long.  Can you put a comment why this is added here?

Comment (coleenp) [reply src/hotspot/cpu/aarch64/frame_aarch64.cpp:166]: I see why it's added here.  Still have a comment why this is.

**Review by coleenp — APPROVED** [review]
Looks like you had a great code walkthrough.

**Review by robehn — APPROVED** [review]

**Review by fisk — APPROVED** [review]
Looks good.

**Pull request closed without merging**